master/getting-started/openstack/requirements.md

```diff
@@ -12,10 +12,20 @@ We aim to develop and maintain the Neutron driver for {{site.prodname}}
 previous release (back to Icehouse), on any operating system, independently of
 the deployment mechanism that is used to install it.
 
-However, our active support and testing of {{site.prodname}} {{page.version}} 
-with OpenStack is limited to the following versions:
+However, we recommend using OpenStack Liberty or later, and our active support
+and testing of {{site.prodname}} {{page.version}} with OpenStack is limited to
+the following versions:
 
 - Ocata
 - Pike
 
-{% include {{page.version}}/reqs-kernel.md %}
\ No newline at end of file
+{% include {{page.version}}/reqs-kernel.md %}
+
+## Nova patch needed with Mitaka and earlier
+
+With OpenStack Mitaka and earlier, and if your libvirt is >= 1.3.3 and < 3.1,
+you will need to patch the Nova code post installation, on each compute host,
+as in [this change](https://review.openstack.org/#/c/411936/).  In case you
+need the same Nova code to work with all possible libvirt versions, you should
+then add [this further change](https://review.openstack.org/#/c/448203/).
+OpenStack Newton and later already include these two changes.
```

Comment:
OK....I think we have this note here: https://docs.projectcalico.org/master/getting-started/openstack/installation/redhat ?

Reply:
Maybe we could just have it in one place. If you want to put it here, I think it would be better as a "Tip" or a "Note". We could also potentially remove it from the Red Hat page? But let me know your thoughts.